starter/js/atm.js

```diff
@@ -1,43 +1,100 @@
 //Begin with the document ready function
-
+$(document).ready(function(){
+  var amountChecking;
+  var checkingBalanceTemp;
+  var checkingBalance;
+  var withdrawChecking;
+
+  var amountSavings;
+  var savingsBalanceTemp;
+  var savingsBalance;
     //Checking account deposit function
+    //On click of the depositChecking button
+  $('#depositChecking').click(function(){
+    //Get value from the amountChecking input field
 
-      //On click of the depositChecking button
-
-        //Get value from the amountChecking input field
 
-        //Take that value and add it to the existing value in the checkingBalance div
+    amountChecking = parseInt($('#amountChecking').val());
+    checkingBalanceTemp = $("#checkingBalance").text(); // get the "$0", you cannot add or substract it directly
+    checkingBalance = parseInt(checkingBalanceTemp.split("").slice(1).join("")); // give you the integer 0
 
-    //Checking account withdrawl funtion
+     //Take that value and add it to the existing value in the checkingBalance div
+    checkingBalance = amountChecking + checkingBalance;
 
-      //On click of the withdrawChecking button
+    $("#checkingBalance").text('$'+checkingBalance);
 
-        //Get value from the amountChecking input field
+    // text() get the value from certain element
+    // text(something) set the value for certain element
 
-        // If that value is greater than the value in the account ignore that action
-        // In other words if this would put the account into a negative balance do not allow it
+  });
 
-        //Else subtract that value from the current amount in the checking account
+    //Checking account withdrawl funtion
+    //On click of the withdrawChecking button
+  $('#withdrawChecking').click(function(){
+    //Get value from the amountChecking input field
+    amountChecking = parseInt($('#amountChecking').val());
+    checkingBalanceTemp = $("#checkingBalance").text(); // get the "$0", you cannot add or substract it directly
```

Comment:
If you want, you can do this all in one move by saying `$("#checkingBalance").text().split("").slice(1).join("")`;